src/vs/workbench/contrib/quickaccess/browser/commandsQuickAccess.ts

```diff
@@ -72,7 +72,13 @@ export class CommandsQuickAccessProvider extends AbstractEditorCommandsQuickAcce
 			noResultsPick: {
 				label: localize('noCommandResults', "No matching commands"),
 				commandId: ''
-			}
+			},
+			suggestedCommandIds: new Set([
+				'workbench.action.tasks.runTask',
+				'workbench.action.files.openFolder',
+				'remoteHub.openRepository',
+				'workbench.action.remote.showMenu'
```

Comment:
In draft until we have the correct list of Command Ids to suggest. cc @digitarald